scripts/plot_benchmarks.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "matplotlib",
+#     "pyqt6",
+#     "numpy",
+# ]
+# ///
+
+# This program shows `hyperfine` benchmark results as a bar plot grouped by command.
+
+import argparse
+import json
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument(
+    "files", nargs="+", type=pathlib.Path, help="JSON files with benchmark results"
+)
```

Comment:
Maybe we could add a note here that all files must include results with the same commands? Or maybe that should be described in the top-level doc string?